### DISCUSSION TITLE
首页内容提议

### DISCUSSION THREAD

**Discussion by fanux** [labring/sealos#1210] [General] [answered]
##  Part 1 一句话介绍

sealos
kubernetes-kernel-based cloud os! Let's sealos run kubernetes

## Part 2 三大特性

简单/灵活/强大

## Part 3 例子

如这种 安装 kubernetes minio mysql redis pgsql openebs 等例子
<img width="879" alt="image" src="https://user-images.githubusercontent.com/8912557/175988124-d5750941-fd2e-4cde-9842-402d10729e4c.png">

## Part 4 谁在使用 sealos 

客户公司 logo ...

## Kernel 架构介绍

左边：
<img width="954" alt="image" src="https://user-images.githubusercontent.com/8912557/176395633-3a743abc-c28c-40cd-b0f8-0592f3df55ba.png">

右边：
* cloud driver - CRI CNI CSI 充当云操作系统的驱动，把所有计算存储网络资源池化，如 containerd calico openebs 等技术
* cloud kernel - kubernetes 是云操作系统的云内核，抽象底层资源，定义资源接口，标准化应用管理 API
* distributed applications -  云内核之上皆为应用，通过应用自由组合满足各种不同需求和场景

## 页脚

公司介绍链接，laf 链接， 联系方式...

**Comment by SignorMercurio** [comment]
Reference for Part3: https://docusaurus.io/zh-CN/docs/markdown-features/tabs

**Comment by fanux** [comment]
场景，sealos 能干啥：

* kubernetes 生命周期
* 高可用数据库，消息队列
* AI 深度学习

**Reply by whybeyoung** [reply]
> 场景，sealos 能干啥：
> 
> 
> 
> * kubernetes 生命周期
> 
> * 高可用数据库，消息队列
> 
> * AI 深度学习


ai推理

**Comment by zzjin** [comment, marked as answer]
<img width="921" alt="cloud-kernel-arch-center" src="https://user-images.githubusercontent.com/679757/176616585-b41c47d9-0795-4ecd-996c-491df8b9300d.png">

fix kernel arch centered image.
